components/layout/block.rs

```diff
@@ -1491,7 +1491,7 @@ impl BlockFlow {
         debug_assert!(!self.is_inline_flex_item());
 
         // Compute the available space for us, based on the actual floats.
-        let rect = self.base.floats.available_rect(Au(0),
```

Comment:
Oh, right. Could we add a test that exercises this? Also, I've got another question, why only reflow and not also `REPOSITION`? There's a long time since I looked at this code, but seems that it could also be needed?